raw-cms-app/src/modules/core/components/top-bar/top-bar.tpl.html

```diff
@@ -3,5 +3,11 @@
     <v-app-bar-nav-icon @click.stop="toggleDrawer()"></v-app-bar-nav-icon>
     <span class="hidden-sm-and-down">Raw CMS</span>
   </v-toolbar-title>
+
```

Comment:
The title does not seem to be properly centered:
![image](https://user-images.githubusercontent.com/10808004/71238455-5fa96600-2304-11ea-8e1d-dc636b86a7bf.png)
Try to use a `<v-toolbar-title>` instead of `<v-toolbar-items>` and remove the `<h1>` tag.

Reply:
use something like this:
```html
<v-spacer></v-spacer>
  <v-toolbar-title>
    {{Title}}
  </v-toolbar-title>
<v-spacer></v-spacer>
```
for the centered title